toolchain/driver/driver.cpp

```diff
@@ -74,6 +78,34 @@ auto Options::Build(CommandLine::CommandBuilder& b) -> void {
       },
       [&](CommandLine::FlagBuilder& arg_b) { arg_b.Set(&verbose); });
 
+  b.AddStringOption(
+      {
+          .name = "runtimes-cache",
+          .value_name = "PATH",
+          .help = R"""(
+Specify a custom runtimes cache location.
+
+By default, the runtimes cache is located in the `carbon_runtimes` subdirectory
+of `$XDG_CACHE_HOME` (or `$HOME/.cache` if not set). If unable to use either, it
+will be placed in a temporary directory that is removed when the command
+completes. This flag overrides that logic with a specific path.
```

Comment:
```suggestion
completes. This flag overrides that logic with a specific path. It has no effect
if --prebuilt-runtimes is set.
```

This wasn't completely obvious to me, so it seems worth spelling out.